### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by thepatrick**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/thepatrick/ReviewNinja-Welcome/pull/2" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -38,3 +38,5 @@ That’s it! These are the basic features of ReviewNinja to help you get
 started. We hope you enjoy using the tool, and please 
 [let us know](https://github.com/reviewninja/review.ninja/issues/new) 
 how we can make it better.
+
+This is a silly line
```
Comment (thepatrick) [inline ReadMe.md]: This is silly !fix

Comment (thepatrick) [inline ReadMe.md]: No it's not! !fixed

File: ReadMe.md [hunk 2]
```diff
@@ -38,3 +38,5 @@ That’s it! These are the basic features of ReviewNinja to help you get
 started. We hope you enjoy using the tool, and please
```
Comment (thepatrick) [inline ReadMe.md]: Testing

**Pull request merged**